core/modules/memman/MemMan.cc

```diff
@@ -0,0 +1,43 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
```

Comment:
Including a .h first in its corresponding .cpp ensures the .h properly pulls in all its own dependencies and can stand alone.  This helps flush out "this was only compiling because somebody included that first, which included that which uncluded that, now somebody else changed their header and i'm busted".  Also ensures your header clients won't have to go hunting to see what they have to include first for your thing to work, etc?